scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/SchedulerHandler.java

```diff
@@ -59,6 +59,11 @@ public SchedulerOutput handleRequest(SchedulerInput input, Context context) {
 
     String activeEnvironmentId = environment.getActiveEnvironmentRevisionId();
```

Comment:
As a suggestion, we can change this into activeEnvironmentRevisonId to avoid confusion

Reply:
Oh, whoops totally a typo, will fix.